packages/frontend/src/components/dashboard/TemplateTable/Container.vue

```diff
@@ -105,6 +105,20 @@ const isAnalyzing = computed(() => {
   return analysisService.jobState.type === "Analyzing";
 });
 
+const defaultPorts = [80, 443];
+
+const getURLColumnValue = (template: TemplateDTO) => {
+  let url = `${template.meta.method} ${template.meta.isTls ? 'https' : 'http'}://${ template.meta.host }`;
+  if (!defaultPorts.includes(template.meta.port)) {
+    url += `:${template.meta.port}`;
+  }
```

Comment:
Should we take into account the scenario where `port === 80` and `isTls === true` or `port === 443` and `isTls === false`

In that case, `https://google.com/` would be wrong and we'd want to show `https://google.com:80` maybe?

Reply:
Sure! Small edge case but I think it's easy enough to implement. Will make the changes soon

Reply:
Updated!